changelog.d/6567.feature

```diff
@@ -0,0 +1 @@
+Add accessibility labels and hints to the plus buttons on the Home, People, and Rooms tabs.
```

Comment:
One small comment: This should be `pr-6567.feature`, and then the PR includes and empty `File` changelog too. Are you able to fix these and then I'll merge.